docs/HACKING.md

```diff
@@ -273,10 +273,11 @@ Additional flags that can be passed to `hack/run-ci-e2e-test.sh` are
 - `-b` gives an alternative path to the WMCO binary. This option overridden in OpenShift CI.
        When building the operator locally, the WMCO binary is created as part of the operator image build process and
        can be found at `build/_output/bin/windows-machine-config-operator`, this is the default value of this option.
-- `-t` specify the test to run. All tests are run if the option is not used. The allowed options are:
-  - `all` all the tests are run
-  - `basic` creation, network and deletion tests are run
-  - `upgrade` creation, upgrade, reconfiguration and deletion tests are run
+- `-t` specify the test to run. All `maintanence` tests are run if the option is not used. The allowed options are:
+  - `maintanence` all the tests are run aside from the `upgrade-*` tests
```

Comment:
> `all` all the tests are run

I think, keeping  `all` makes more sense.

Reply:
happy to keep `all` if that's the consensus, though it is a bit of a misnomer since it doesn't run the upgrade tests

Reply:
all is a misnomer.
Why not replace it with basic? Don't see why we need basic if its not being used in CI.